tedana/utils.py

```diff
@@ -108,6 +124,15 @@ def make_adaptive_mask(data, mask=None, getsum=False, threshold=1):
     # and count # of echos that pass criterion
     masksum = (np.abs(echo_means) > lthrs).sum(axis=-1)
 
+    # Determine where voxels stop decreasing in signal from echo to echo
```

Comment:
Please give feedback on the proposed method.

Reply:
Also, would it make sense to use this method _instead of_, rather than in addition to, the current method?

Reply:
The method looks fine. I'd keep both. Even if it's slightly arbitrary, the original method is an actual measure of dropout. The new method should only find voxels with substantial dropout and voxels where there's something wrong with the data.

Reply:
We still do need to update the original method to reflect the _last_ good echo (see https://github.com/ME-ICA/tedana/issues/679#issuecomment-1956766801).